iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningTests.java

```diff
@@ -81,19 +60,19 @@ public void enrollmentGroupRegistration() throws Exception
         basicRegistrationFlow(EnrollmentType.GROUP);
     }
 
-    @Ignore // The DPS instance we use for this test is currently offline, so this test cannot be run
     @Test
     @ContinuousIntegrationTest
     public void individualEnrollmentWithInvalidRemoteServerCertificateFails() throws Exception
     {
+        Assume.assumeTrue(System.getProperty("os.name").toLowerCase().contains("linux"));
```

Comment:
duplicated logic for how to check if the test is run on Linux. See my other comment about factoring this out somewhere